examples/user_guide/03-Applying_Customization.ipynb

```diff
@@ -450,9 +444,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Passing `hv.output` an object will apply the specified settings only for the subsequent display. If you were to view `path` now in the usual way, you would see that it is still being displayed with Bokeh with red dotted lines.\n",
+    "Passing `hv.output` an object will apply the specified settings only for the subsequent display. If you were to view `path` now in the usual way, you would see that it is still being displayed with Bokeh with purple dotted lines.\n",
     "\n",
-    "One thing to note is that when we set the options with `backend='matplotlib'`, the active plotting extension was Bokeh. This means that `opts.Path` will tab completes *bokeh* keywords and not the matplotlib ones that were specified. However, as matplotlib was loaded with `hv.extension`, these matplotlib keywords are considered valid regardless."
+    "One thing to note is that when we set the options with `backend='matplotlib'`, the active plotting extension was Bokeh. This means that `opts.Path` will tab complete *bokeh* keywords, and not the matplotlib ones that were specified. However, as matplotlib was loaded with `hv.extension`, these matplotlib keywords are considered valid regardless. In practice you will want to set the backend appropriately before building your options settings, to ensure that you get the most appropriate tab completion."
```

Comment:
I agree with the last comment but it does undercut the point of having the output cell magic equivalent...

Reply:
Sure, but I don't think that's a problem.